src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Views/EmployerRequest/SelectRequestsToContact.cshtml

```diff
@@ -0,0 +1,149 @@
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_ValidationSummary" />
+            <h2 class="govuk-caption-xl">
+                @Model.StandardTitle (level @Model.StandardLevel)
+            </h2>
+            <h1 class="govuk-heading-xl">@ViewData["Title"]</h1>
+            <p class="govuk-text">
+                If you're able to offer the course, we'll share your contact details with the employers you select.  
+                The employers name and contact details are not shown in the requests.
+            </p>
+            <p class="govuk-text">
+                You can also view the requests from employers you've already contacted.
+            </p>
+
+        </div>
+        <div class="govuk-grid-column-full">
+            <p class="govuk-body">
+                @Model.AllEmployerRequests.Count()<text> results </text>
+            </p>
+            <!-- tabs -->
+            <div class="govuk-tabs govuk-grid-column-two-thirds" data-module="tabs">
+                <ul class="govuk-tabs__list">
+                    <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+                        <a class="govuk-tabs__tab" href="#notcontacted">
+                            Not Contacted (@Model.AllEmployerRequests.Count() results)
+                        </a>
+                    </li>
+                    <li class="govuk-tabs__list-item">
+                        <a class="govuk-tabs__tab" href="#contacted">
+                            Contacted (0 results)
+                        </a>
+                    </li>
+                </ul>
+                <section class="govuk-tabs__panel" id="notcontacted">
+                    <div id="not-contacted-requests">
+                        <form asp-route="@EmployerRequestController.SelectRequestsToContactRoutePost" method="post">
+                            @Html.HiddenFor(model => model.Ukprn)
+                            @Html.HiddenFor(model => model.StandardReference)
+                            @if(Model.AllEmployerRequests.Any())
+                            {
+                                <div class="govuk-form-group" das-highlight-error-for="SelectedRequests"
+                                 error-class="govuk-form-group--error">
+                                    <fieldset class="govuk-fieldset">
+                                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                                            Which requests for @Model.StandardTitle (level @Model.StandardLevel) do you want to respond to?
+                                        </legend>
+                                        <div id="SelectedRequests-hint" class="govuk-hint">
+                                            Select all that apply.
+                                        </div>
+                                        <div class="govuk-form-group" esfa-validation-marker-for="SelectedRequests">
```

Comment:
I don't think esfa-validation-marked is doing anything and asp-validation-for - should be das-validation-for

Reply:
Updated tags